job_creator.php

```diff
@@ -374,7 +375,7 @@ private function getListOfPhpVersionsByBranchName(): array
         // Fallback to using a CMS major based on the version of PHP in composer.json
         $json = $this->getComposerJsonContent();
         if ($json) {
-            $php = $json->require->php ?? null;
+            $php = $json->require->php ?? '';
```

Comment:
Fix deprecation warning

`Deprecated: str_replace(): Passing null to parameter #3 ($subject) of type array|string is deprecated in /path/to/gha-generate-matrix/job_creator.php on line 379`